### PR TITLE
Pull request for MSLQ-81

### PR DESCRIPTION
**Pull request by exoticDFT**
Updates the companion computer section documentation for the quick start guide.

### CODE REVIEW

**Review by harukins — COMMENTED** [review]
Looks good to me. Some refs are shown like in this picture on my phone. I'm not sure if this is a GitHub thing or rst native.
Screenshot_20201109-231846

File: docs/source/software/quick_start_guide.rst [hunk 1]
```diff
@@ -50,17 +50,12 @@ Companion Computer Setup
...
-1. Odroid
-2. Upboard
-3. TX2
-
-.. TODO:
-    Link to cloned image for those not in lab.
-
-Borrow the relevant USB stick from the lab.
+Instructions for flashing can be found
+:ref:`here<software/Companion_computer:Companion Computer>`.
```
Comment (k2shah) [inline docs/source/software/quick_start_guide.rst:58]: i think you need a newline here? Before `:ref:`

Comment (exoticDFT) [reply docs/source/software/quick_start_guide.rst:58]: I will double check this and see what's going on. My guess, is I missed the correct spelling/capitalization of the link. When you saw this, was it with your local build? Did you see any errors or warnings in the output?

Comment (exoticDFT) [reply docs/source/software/quick_start_guide.rst:58]: I currently cannot reproduce this on my machine.

It seems to me this is a Sphinx environment issue... I added the extension `sphinx.ext.autosectionlabel` in order to generate labels for all the section headers automagically. This required a change to the `conf.py` and most likely will require a `make clean && make html` locally in order to see the correct functionality. But let me know if you've done this and are still seeing the issue. If so, there still may be something I'm doing on my machine that allows this to function as I was hoping and I haven't provided it to the repo.

Comment (k2shah) [reply docs/source/software/quick_start_guide.rst:58]: seems resolved now. super minor but
`Image of the OS (kessel.stanford.edu/media/data/odroid_images/20190305_MSL_Quad_ODroidKernel4_14_Ubuntu16_04_Rootless_GPIO.img`

extends off the page. change to 
the image of the OS can be found on `kessel` at 
`kessel.stanford.edu/media/data/odroid_images/`
with the file name
`20190305_MSL_Quad_ODroidKernel4_14_Ubuntu16_04_Rootless_GPIO.img`

Comment (exoticDFT) [reply docs/source/software/quick_start_guide.rst:58]: Great catch. Updating this and then merging!

**Pull request merged**